device/samples/helpers/device-twin-service/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "iot-hub",
```

Comment:
>"name": "iot-hub", [](http://example.com/codeflow?start=2&length=18)

You should give this a better name and add it to lerna.json and the inputs.js file in the internals repo.  lerna.json ensures that the dependencies get linked for local development, and input.js ensures that the dependency versions get updated.

Reply:
@BertKleewein I updated the name and added to lerna.json. I can't find an inputs.js file